### DISCUSSION TITLE
Unimplemented syscalls

### DISCUSSION THREAD

**Discussion by gandalfthewhite19890404** [obhq/obliteration#243] [General]
Should I create separate issue topic with unimplemented syscalls?
I mean - I do test, I see some unimplemented syscall. I edit my post in issue by adding syscall, after implementing I'll mark it as completed, redo a test and if I'll get new syscall I will add it again to initial post etc.

**Comment by ultimaweapon** [comment]
It will be great if you can help with that :)

**Comment by gandalfthewhite19890404** [comment]
Done!

**Reply by ultimaweapon** [reply]
Thanks!

**Comment by ultimaweapon** [comment]
BTW do you interested to do a reverse engineering on the PS4 kernel and help on implementing the syscalls? You need to know C in order to do this. I can give some starting guides for this if you are interested.
